tests/Microsoft.Identity.Test.Android.UIAutomation/androidTests.cs

```diff
@@ -91,12 +91,12 @@ public void RunAllTests()
                 AcquireTokenADFSV4InteractiveFederatedTest,
                 AcquireTokenADFSV4InteractiveNonFederatedTest,
 
-                //B2CFacebookB2CLoginAuthorityAcquireTokenTest,
-                //B2CFacebookMicrosoftAuthorityAcquireTokenTest,
+                B2CFacebookB2CLoginAuthorityAcquireTokenTest,
```

Comment:
in dev3x, the ios tests are not passing. there is an issue w/the app. but the android FB tests are now passing. #fridaynightissues

Reply:
fantastic. we need to release stuff to avoid this situation

---
In reply to: [251188978](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/839#discussion_r251188978) [](ancestors = 251188978)